scripts/createComponent.js

```diff
@@ -0,0 +1,233 @@
+/* eslint-disable */
+const path = require('path');
+const fs = require('fs');
+
+const replacementString = 'Component';
+const componentPath = 'common/components';
+
+
+/*
+ *  Modify these strings to take the componentName string and build any values you need. 
+ */
+const buildStoryJs = (componentName) => 
+  `import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { withKnobs, boolean, select, text } from '@storybook/addon-knobs';
+
+import ${componentName} from '../${componentName}';
+
+storiesOf('Common/${componentName}', module)
+  .addDecorator(withKnobs)
+  .add(
+    'default',
+    withInfo()(() => (
+      <${componentName}
+      id={text('id', 'heading1')}
+      has${componentName}Lines={boolean('has${componentName}Lines', true)}
+      theme={select('theme', ['gray', 'slate', 'white'], 'gray')}
+        >
+        {text('children', '${componentName} Component')}
+        </${componentName}>
+    )),
+  );
+`;
+
+
+const buildTestJs = (componentName) => 
+  `
+/* eslint-env jest */
+import React from 'react';
+import { shallow } from 'enzyme';
+import createSnapshotTest from 'test-utils/createSnapshotTest';
+
+import ${componentName} from '../${componentName}';
+
+describe('${componentName}', () => {
+  test('should render with just required props passed', () => {
+    createSnapshotTest(<${componentName}>Test</${componentName}>);
+  });
+
+  test('should render properly with some props assigned', () => {
+    createSnapshotTest(
+      <${componentName} className="test-class" id="test-heading-1" has${componentName}Lines={false}>
+        Test
+        </${componentName}>,
+    );
+  });
+
+  test('should render with "slate" in classNames when theme="slate"', () => {
+    const HeaderInstance = shallow(<${componentName} theme="slate">Test</${componentName}>);
+
+    expect(HeaderInstance).toHaveClassName('slate');
+  });
+
+  test('should render without "headingLines" in classNames when has${componentName}Lines={false}', () => {
+    const HeaderInstance = shallow(<${componentName} has${componentName}Lines={false}>Test</${componentName}>);
+
+    expect(HeaderInstance).not.toHaveClassName('headingLines');
+  });
+});
+`
+
+
+const buildCss = (componentName) => `.${componentName} {
+
+  }>`;
```

Comment:
Replace the `>` with `\n` and you can make this one whole function line 👍 - it's okay that it'll end up like that in the file.